decidim-core/config/locales/en.yml

```diff
@@ -3,6 +3,8 @@ en:
     attributes:
       account:
         delete_reason: Reason to delete your account
+      amend:
+        user_group_id: Amendment author
```

Comment:
Shouldn't the key be something like `author`?

Reply:
I think this have to do with letting the amender choose to author the amendment as a user or as a user group.
But in our team we also discussed that in the control versions the author of the version should always be the user that created the amendment and not the user group (if chosen) to avoid confusions.
So maybe letting the amender make that choice in the create from isnt a good idea. What do you think?

Reply:
I mean that the i18n key should be `amendment_author`, not `user_group_id`, since the latter is misleading.

Reply:
We addressed the issue with the team once more and concluded that we should respect the choice of the amender to author the amendment as user/user group and reflect it in the control versions. 
So yes, since this key will be used it should be amendment_author to not be misleading. Thanks